src/components/layout/work/work-card.tsx

```diff
@@ -17,16 +17,31 @@ const WorkCard: React.FC<WorkCardProps> = ({
   slug,
 }) => {
   return (
-   <div className="relative group rounded-2xl overflow-hidden shadow-lg w-full h-80">
-    <img src={image} alt={title} className="absolute inset-0 w-full h-full object-cover group-hover:scale-110 transition-transform duration-500" />
-    <div className="absolute inset-0 bg-gradient-to-t from-black via-transparent to-transparent opacity-0 group-hover:opacity-100 transition-opacity duration-500">
-        <div className="absolute bottom-4 left-4">
-          <h3 className="text-white text-lg font-semibold">{title}</h3>
-          <p className="text-gray-300 text-sm">{description}</p>
-        </div>
+    <div className="relative group rounded-2xl overflow-hidden shadow-lg w-full h-80">
+
+  <img
+    src={image}
+    alt={title}
+    className="absolute inset-0 w-full h-full object-cover group-hover:scale-110 transition-transform duration-500"
+  />
+
```

Comment:
```suggestion
        <img
          src={image}
          alt={title}
          className="absolute inset-0 w-full h-full object-cover group-hover:scale-110 transition-transform duration-500"
        />
```

Lütfen formatting'e dikkat edelim